### PR TITLE
PLAT-143713: Updated screenshot tests for ProgressBar

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added screenshot tests for RTL locale for ProgressBar

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143713

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

File: tests/screenshot/apps/components/ProgressBar.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import ProgressBar, {ProgressBarTooltip as Tooltip} from '../../../../ProgressBar';
 
-// TODO: RTL
+import {withConfig} from './utils';
+
 // [GT-28227] - generic to test Progress Bar and Background Progress Bar Colors - no end marked for this test
-const ProgressBarTests = [
+const ProgressBarLTRTests = [
 	<ProgressBar />,
 	<ProgressBar highlighted />,
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/ProgressBar.js:8]: Can you add a test for disabled as well?

Comment (daniel-stoian-lgp) [reply tests/screenshot/apps/components/ProgressBar.js:9]: 👍

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**